### PR TITLE
Issue #7060: aligned javadoc/xdoc for MethodCount

### PR DESCRIPTION
**Pull request by pbludov**
Issue #7060 (part of #5750)

No changes in the code.

### CODE REVIEW

File: src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java [hunk 1]
```diff
@@ -97,9 +97,6 @@
         "CyclomaticComplexity",
         "JavaNCSS",
         "NPathComplexity",
-        // sizes
-        "FileLength",
```
Comment (rnveach) [inline src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: Why is this removed?

Comment (pbludov) [reply src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java]: I forgot to remove this in #7073

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodCountCheck.java [hunk 2]
```diff
@@ -32,8 +32,107 @@
...
+ *     });
+ *   }
+ *
+ *   public static class InnerExampleClass {
+ *     protected void example2() { ... } // NOT counted towards ExampleClass,
+ *                                    // but counted towards InnerExampleClass
+ *   }
+ * }
+ * </pre>
+ * </p>
```
Comment (rnveach) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodCountCheck.java]: > Exit code: 1 - /home/travis/build/checkstyle/checkstyle/src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodCountCheck.java:74: error: unexpected end tag: </p>

Comment (pbludov) [reply src/main/java/com/puppycrawl/tools/checkstyle/checks/sizes/MethodCountCheck.java]: done
